src/Emitter/SapiEmitterTrait.php

```diff
@@ -79,8 +79,9 @@ private function emitHeaders(ResponseInterface $response): void
         $statusCode = $response->getStatusCode();
 
         foreach ($response->getHeaders() as $header => $values) {
+            assert(is_string($header));
             $name  = $this->filterHeader($header);
-            $first = $name === 'Set-Cookie' ? false : true;
+            $first = ! ($name === 'Set-Cookie');
```

Comment:
I think this is easier to read:

```suggestion
            $first = $name !== 'Set-Cookie';
```

Reply:
Yeah, phpstorm suggestion 😅